core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -206,32 +269,38 @@
   /**
    * Handles a netty RPC callable with logging.
    *
-   * @param logger the logger
-   * @param callable the netty RPC callable
-   * @param <T> the return type
+   * @param logger the logger to use for this call
+   * @param callable the callable to call
+   * @param methodName the name of the method, used for metrics
+   * @param description the format string of the description, used for logging
+   * @param args the arguments for the description
+   * @param <T> the return type of the callable
    * @return the rpc result
    */
-  public static <T> T nettyRPCAndLog(Logger logger, NettyRPCCallable<T> callable) {
-    logger.debug("Enter: {}", callable);
-    try (Timer.Context ctx = MetricsSystem.timer(getQualifiedMetricName(callable)).time()) {
+  public static <T> T nettyRPCAndLog(Logger logger, NettyRpcCallable<T> callable,
+      String methodName, String description, Object... args) {
+    // avoid string format for better performance if debug is off
+    String debugDesc = logger.isDebugEnabled() ? String.format(description, args) : null;
+    try (Timer.Context ctx = MetricsSystem.timer(getQualifiedMetricName(methodName)).time()) {
+      logger.debug("Enter: {}: {}", methodName, debugDesc);
       T result = callable.call();
-      logger.debug("Exit (OK): {}", callable);
+      logger.debug("Exit (OK): {}: {}", methodName, debugDesc);
       return result;
     } catch (Exception e) {
-      logger.debug("Exit (Error): {}, Error={}", callable, e.getMessage());
-      MetricsSystem.counter(getQualifiedFailureMetricName(callable)).inc();
+      logger.warn("{}: {}, Error={}", methodName, String.format(description, args), e);
```

Comment:
ditto

Reply:
I'm leaving out the prefix for these because there is not a corresponding `(Enter)`

Reply:
but in debug mode, it will be strange becuase it has `Enter`. How about

```java
if (logger.isDebugEnabled) {
  // debug style logging
} else {
  // warn logging
}
```

Reply:
I would prefer to always just have `Exit (Error)` instead of adding more complexity in this part of the code for relatively little gain.